.eslintrc.cjs

```diff
@@ -4,18 +4,18 @@ const process = require('process');
 
 const lintTypes = !!process.env.AGORIC_ESLINT_TYPES;
 
-const notLoanDeprecated = [
```

Comment:
Why rename this? The intent was to be the complement of "loan" in allDeprecated

Reply:
Yeah, that's what I thought it should be when I introduced `notLoanDeprecated`, but I found it weird to have an override targeting the loan contract and saying to use a `notLoan` rule.  I thought it would make more sense to have `loanContractDeprecated`, and then a superset of that for `allDeprecated`.

Maybe better names would be `deprecatedForLoanContract` and `deprecatedForAll`?

Reply:
`deprecatedForLoanContract` doesn't sound like the subset it is. but this is fine. IMO the best solution is to move the sample code somewhere else and that's out of scope.